packages/design-system/src/components/Dialog/Dialog.tsx

```diff
@@ -123,31 +123,26 @@ export const Dialog = (props: DialogProps) => {
       {...modalProps}
       id={rootId}
       boundingBoxRef={containerRef}
+      aria-labelledby={headingId}
     >
-      <div
-        className="ds-c-dialog__window"
-        role="document"
-        ref={containerRef}
-        tabIndex={-1}
-        aria-labelledby={headingId}
-      >
-        <header className={headerClassNames}>
+      <div className="ds-c-dialog__window" ref={containerRef} tabIndex={-1}>
+        <div className={headerClassNames}>
```

Comment:
I just know the original design was tested by accessibility specialists with a variety of screen readers, so will this be tested again by the accessibility team before merging?

Reply:
@TiffanyPender would it be possible to get another review of the [Dialog changes](https://cmsgov.github.io/design-system/branch/WNMGDS-2712/update-deprecated-aria/storybook/?path=/story/components-dialog--dialog-example&globals=theme:core) in this PR? 

The changes made to this component are:
- Moving `aria-labelledby` to the `<dialog>` element
- Replacing the `<header>` and `<main>` tags with `<div>`
    - `<header>` and `<main>` didn't have accessible names
    - Including `<main>` seemed to fly against the rule of only 1 `<main>` element / page
- Removing `role="document"` from an internal `<div>`

These changes would align Dialog with Drawer markup (Drawer does not use these elements) and I have a sneaking suspicion the original Dialog markup was written well before the release of `<dialog>`, so attributes like `role="document"` are unnecessary.

Reply:
@pwolfert I got feedback [from Tiffany with feedback for Dialog](https://jira.cms.gov/browse/WNMGDS-2796).

Changes introduced:
1. Moved focus to `<dialog>` element (where the `aria-labelledby` can label the contents of the dialog)
2. Added the option to hide aria content for CloseButton